content/en/blog/2021-03-19-a-parrot-trainer-eats-crow.md

```diff
@@ -0,0 +1,100 @@
+
+The progression over the last few years has been amazing to watch. There have never been more excellent open source resources for people who do what I do. It has never been easier to [bootstrap a domain-specific language model](https://rebeccabilbro.github.io/tailored-learning/), even if [you don't have much data to start with](https://rebeccabilbro.github.io/small-data-delegated-literacy/). But it's also true that we have never been more removed from our data than we are today, or less in touch with its underlying patterns, themes, and biases.
+
+This problem is at the heart of the recent paper [On the Dangers of Stochastic Parrots: Can Language Models Be Too Big?](https://dl.acm.org/doi/pdf/10.1145/3442188.3445922) The paper itself is at the heart of a controversy about Google's [abrupt dismissal](https://www.bbc.com/news/technology-56135817) of two of the authors, Dr. Timnit Gibru and Dr. Margaret Mitchell, who helped found and lead Google's AI Ethics team.
+
+The Parrots paper discusses a range of concerns with large language models (like the ones Google makes), including the dangers of "ersatz fluency" and the environmental costs of training such models. Indeed, we seem to be entering a new phase in which machine models are only distinguishable from humans by their absence of legal and ethical responsibilities for the ramifications of their words and actions. Moreover, as with cryptocurrencies and [cryptoart like NFTs](https://everestpipkin.medium.com/but-the-environmental-issues-with-cryptoart-1128ef72e6a3), it is becoming clear that the costs are disproportionately paid by people unlikely to realize much of their benefits.
+
+The paper is, however, primarily a warning about the challenge of responsibly building deep learning models that require a volume of data that exceeds human capacity to effectively curate. And as an NLP developer, this was the part of the paper that triggered that uneasy feeling in the pit of my stomach. This is something that ~~we~~ I need to take responsibility for and help fix. But how?
```

Comment:
Let's be sure to check that our website renders the strikethrough markdown when we publish!

Reply:
Checked - it does!